scylla-cdc/src/cdc_types.rs

```diff
@@ -0,0 +1,42 @@
+use scylla::cql_to_rust::{FromCqlVal, FromCqlValError};
```

Comment:
About the commit message: "Add cdc reader component by provided stream ids" what does it even mean?

Reply:
I have changed the commit message to include more details, however, if you have other suggestions feel free to tell them.